src/server/utilities/cspHeader/index.js

```diff
@@ -12,64 +12,41 @@ import { bbcDomains, advertisingServiceCountryDomains } from './domainLists';
 const advertisingDirectives = {
   connectSrc: [
     'https://csi.gstatic.com',
-    'https://experience.tinypass.com',
-    'https://pagead2.googlesyndication.com',
-    'https://*.g.doubleclick.net',
-    'https://survey.effectivemeasure.net',
-    'https://collector.effectivemeasure.net',
-    'https://detect-survey.effectivemeasure.net',
+    'https://*.googlesyndication.com',
+    'https://*.doubleclick.net',
+    'https://*.effectivemeasure.net',
     'https://adservice.google.com',
-    'https://tpc.googlesyndication.com',
-    'https://ad.doubleclick.net',
-    'https://googleads.g.doubleclick.net',
-    'https://securepubads.g.doubleclick.net',
+    'fundingchoicesmessages.google.com',
```

Comment:
I'm not completely certain but I suspect you need `https://` adding to the start of these domains for this change to work properly?

Reply:
https://experience.tinypass.com is not in use on WS sites
*crwdcntrl.net that's a third party that has been deprecated

I've also removed some instances like these
'https://googleads.g.doubleclick.net',
'https://securepubads.g.doubleclick.net'
As they are already covered by 'https://*.g.doubleclick.net',
Ideally we'd have all the separate instances but that's a tough one to get :(

Hope this helps

Added
'https://fundingchoicesmessages.google.com', // new CMP launching soon
    'https://secure-dcr-cert.imrworldwide.com', // something that is currently missing
    'https://pixel.adsafeprotected.com', // new ad safety launching soon
    'https://cdn.adsafeprotected.com', // new ad safety launching soon